app/models/messaging_preference.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 require 'common/models/base'
-require 'evss/pciu/email_address'
+require 'va_profile/models/email'
```

Comment:
Models should be autoloaded - making this require unnecessary. If it's not for some reason, we may need to define an inflection, but the [VA inflection should take care of this ](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/initializers/inflections.rb#L36)

Reply:
@LindseySaari Looking now! Thanks